templates/cloudwatch-rds-postgresql.yml

```diff
@@ -26,8 +26,8 @@ Parameters:
     Description: Sample rate. See https://honeycomb.io/docs/guides/sampling/.
   LogLinePrefix:
     Type: String
-    Default: '%t [%p-%l] %u@%d'
-    Description: The prefix of each log line. See https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-LINE-PREFIX
+    Default: '%t:%r:%u@%d:[%p]:t'
+    Description: The prefix of each log line. See https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.PostgreSQL.html#USER_LogAccess.Concepts.PostgreSQL.Log_Format
```

Comment:
Suggestion - since this should never be changed, why give users the option?  I can't think of any reasons why this should be in the parameters list at all.

Reply:
In case that AWS policy changes at some point?

Also, is there any impact on current users of the template if the parameter is removed?